python/cdo.py

```diff
@@ -353,6 +353,7 @@ def __loadOptionalLibs(self):
       print("-->> Could not load netCDF4! <<--") #}}}
 
   def infile(self, *infiles):
+    # if this function still used?
```

Comment:
oh, i guess, this is used for operator chaining when no input argument is avaiable? sorry for the stupid comment!

Reply:
hi! sorry for being so late.
I see the need for mixed type inputs, but could you come up with a test that shows the new feature? It's hard to work on something without the right feeling for a use case. tests will ensure this will work not only today but tomorrow

cheers + healthy summer

Reply:
ok, you are right, i'll add a test!